### ISSUE TITLE
A paragraph that lacks context

### ISSUE THREAD

**Issue by ghaya** [LibraryCarpentry/lc-data-intro#152]
In 02-match-extract-strings.md the paragraph below is under the heading "Exercise finding phone numbers..." but I do not see how it relates to that heading.  My suggestion is simply to add a new heading above the paragraph along the lines of "Using regular expressions when working with files and directories"

This is the paragraph in question:
"One of the reasons we stress the value of consistent and predictable directory and filenaming conventions is that working in this way enables you to use the computer to select files based on the characteristics of their file names. For example, if you have a bunch of files where the first four digits are the year and you only want to do something with files from '2017', then you can. Or if you have 'journal' somewhere in a filename when you have data about journals, you can use the computer to select just those files. Equally, using plain text formats means that you can go further and select files or elements of files based on characteristics of the data within those files."

**Comment by libcce** [comment]
Thanks, @ghaya! That's a good catch and we should probably tie that section with lesson material in the workshop overview https://librarycarpentry.org/lc-overview/06-file-naming-formatting/index.html which we moved recently. I like your heading! I think it might be best to have this as a callout for the moment. I've created https://github.com/LibraryCarpentry/lc-data-intro/pull/153 if that looks good to you? I've also pointed to the workshop overview material on file naming and formats.
